osu.Game/Screens/Select/Carousel/CarouselBeatmap.cs

```diff
@@ -47,6 +47,9 @@ public override int CompareTo(FilterCriteria criteria, CarouselItem other)
                     if (ruleset != 0) return ruleset;
 
                     return Beatmap.StarDifficulty.CompareTo(otherBeatmap.Beatmap.StarDifficulty);
+
+                case SortMode.Length:
```

Comment:
unsure about doing this, since we don't do it for any other sort methods yet. i would feel better not having this here just yet, until we figure the UX for second-level sort out.

Reply:
yeah i was thinking about that, fair enough for now